doc/source/ray-observability/ray-logging.rst

```diff
@@ -36,6 +36,26 @@ When logs are printed, the process id (pid) and an IP address of the node that e
 
     (pid=45601) task
 
+Actor log messages look like the following by default.
+
+.. code-block:: bash
+
+    (MyActor pid=480956) actor log message
+
+Customizing Actor logs prefixes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is often useful to distinguish between log messages from different actors. For example, suppose you have a large number of worker actors. In this case, you may want to be able to easily see the index of the actor that logged a particular message. This can be achieved by defining the ``__repr__`` method for an actor class. When defined, the actor repr will be used in place of the actor name. For example:
```

Comment:
Nit: add a link to https://docs.python.org/3/library/functions.html#repr? So that people understand they are overriding a Python built-in method.

Question (not blocking the PR): is this scenario common enough such that we should create something like a superclass called `IndexedActor`?

Reply:
Done. Hmm, I think the use cases have been one-off enough an indexed actor would generally need to be customized anyway.

Reply:
Sounds good. Thanks